### PR TITLE
fix: .gitignore was ignoring files that were needed for website

### PR DESCRIPTION
**Pull request by calebjclark**
Now that tsc is building js files into build/, we can remove some of the .gitignore lines. Doing so highlighted the fact that two needed website js files were being excluded.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -6,32 +6,18 @@ lerna-debug.log
...
 
-!jest-*.js
-!dist-prepare.js
+/build/
+/build-dist/
 
 /mitm/socket/connect
-!mitm/install.js
 /mitm/socket/build/
-/.mitm-ca
```
Comment (blakebyrnes) [inline .gitignore:27]: shouldn't this one still be here?

File: .gitignore [hunk 2]
```diff
@@ -6,32 +6,18 @@ lerna-debug.log
...
-/**/.mitm-ca
 
 emulators/data/DOM-*.json
-!/emulator-plugins/shared/injected-scripts/*.js
-!/emulator-plugins/shared/test/inspectHierarchy.js
-/emulator-plugins/shared/injected-scripts/index.js
-
 
-/.sessions
```
Comment (blakebyrnes) [inline .gitignore:36]: Shouldn't this also still be .gitignored?

File: .gitignore (hunk 1 above)
Comment (calebjclark) [reply .gitignore:27]: Yes. I started the new .secret-agent data dir as part of this same PR, then split.

Will push a fix.

File: .gitignore (hunk 2 above)
Comment (calebjclark) [reply .gitignore:36]: Yep.

**Pull request merged**